src/generate-gateway-key.ts

```diff
@@ -1,85 +1,12 @@
-import { promptQuestions } from './utils/prompts';
-import { readOperationsRepository } from './utils/read-operations';
-import { writeOperationsRepository } from './utils/write-operations';
 
-const questions = (operationsRepository: OperationsRepository): PromptObject[] => {
-  return [
-    {
-      type: 'autocomplete',
-      name: 'name',
-      message: 'What is the name of the API Integration?',
-      choices: Object.keys(operationsRepository.apis).map((api) => ({ title: api, value: api })),
-    },
-    {
-      type: 'autocomplete',
-      name: 'deployment',
-      message: 'For which deployment do you want to generate the gateway keys for?',
-      choices: (prev) =>
-        Object.keys(operationsRepository.apis[prev].deployments).map((deployment) => ({
-          title: deployment,
-          value: deployment,
-        })),
-    },
-  ];
-};
-
-const main = async (operationRepositoryTarget?: string) => {
+const main = async () => {
   const httpGatewayKey = randomBytes(48).toString('hex');
   const signedHttpGatewayKey = randomBytes(48).toString('hex');
 
-  const operationsRepository = readOperationsRepository(operationRepositoryTarget);
-  const response = await promptQuestions(questions(operationsRepository));
-
-  const api = operationsRepository.apis[response.name];
-  const secretAppend = sanitiseFilename(api.apiMetadata.name).toUpperCase() + `_AWS`;
-
-  const oldSecrets = operationsRepository.apis[response.name].deployments[response.deployment].airnode.secrets.content
-    .trim()
-    .split('\n')
-    .filter((secret) => !secret.includes('HTTP_GATEWAY_KEY') && !secret.includes('HTTP_SIGNED_DATA_GATEWAY_KEY'));
-
-  const newSecrets = {
-    filename: '.env',
-    content: [
-      ...oldSecrets,
-      `HTTP_GATEWAY_KEY_${secretAppend}=${httpGatewayKey}`,
-      `HTTP_SIGNED_DATA_GATEWAY_KEY_${secretAppend}=${signedHttpGatewayKey}`,
-    ].join('\n'),
-  };
-
-  const updatedOpsData: OperationsRepository = {
-    ...operationsRepository,
-    apis: {
-      ...operationsRepository.apis,
-      [response.name]: {
-        ...operationsRepository.apis[response.name],
-        deployments: {
-          ...operationsRepository.apis[response.name].deployments,
-          [response.deployment]: {
-            ...operationsRepository.apis[response.name].deployments[response.deployment],
-            airnode: {
-              ...operationsRepository.apis[response.name].deployments[response.deployment].airnode,
-              secrets: newSecrets,
-            },
-            airkeeper: {
-              ...operationsRepository.apis[response.name].deployments[response.deployment].airkeeper,
-              secrets: newSecrets,
-            },
-          },
-        },
-      },
-    },
-  };
-
   console.log(`🔑 Generated HTTP Gateway API Key: ${httpGatewayKey}`);
   console.log(`🔑 Generated HTTP Signed Data Gateway API Key: ${signedHttpGatewayKey}`);
-
-  writeOperationsRepository(updatedOpsData, operationRepositoryTarget);
```

Comment:
Nice! I was actually wondering what was the point of updating the secrets files. That would have probably lead to commit and push secrets to git repo.